kernel/packages/shared/apis/SignedFetch.ts

```diff
@@ -50,7 +60,13 @@ export class SignedFetch extends ExposableAPI {
         ...getAuthHeaders(
           init?.method ?? 'get',
           path,
-          { sceneId: this.parcelIdentity.cid, parcel: this.getSceneData().scene.base, tld: getTLD() },
+          {
+            sceneId: this.parcelIdentity.cid,
+            parcel: this.getSceneData().scene.base,
+            tld: getTLD(),
+            isGuest: isGuest(),
```

Comment:
just to doublecheck, does this method add the timestamp to the signature? to prevent "double spend" attacks

Reply:
It is added above: https://github.com/decentraland/explorer/blob/ab068854fe8739a6705ca7e6a547025ab00e302b/kernel/packages/shared/apis/SignedFetch.ts#L29